src/actions/enter.ts

```diff
@@ -106,6 +107,8 @@ export class EnterAction {
       curNode instanceof WorkspaceSvg
     ) {
       this.navigation.openToolboxOrFlyout(workspace);
+    } else if (curNode instanceof icons.Icon) {
+      curNode.onClick();
```

Comment:
I wonder if this should perhaps focus the icon, instead. Presumably we could then update icons' `onNodeFocus` implementations to their bubbles as needed.

Reply:
The icon is presumably already focused if this is being triggered though, no?

Reply:
Ah you're right--my comment doesn't make sense.

Calling `onClick` does feel a bit weird, though. I suppose having an actual "doAction"-esque function would be a bit cleaner than `onClick` (since this is masquerading as an event provider a bit), but I don't have strong opinions here.